war/src/main/less/modules/table.less

```diff
@@ -0,0 +1,224 @@
+.jenkins-table {
+  width: calc(100% - 10px);
+  background: var(--table-background);
+  border-collapse: collapse;
+  border-radius: 6px;
+  overflow: hidden;
+  box-shadow: 0 0 0 5px var(--table-background);
+  margin: 5px;
+  --table-padding: 0.55rem;
+
+  * {
+    -webkit-border-horizontal-spacing: 0;
+    -webkit-border-vertical-spacing: 0;
+  }
+
+  & > thead {
+    & > tr {
+      & > th {
+        color: var(--table-header-foreground);
+        text-align: left;
+        padding-top: calc((var(--table-padding) * 2) - 5px);
+        padding-bottom: calc(var(--table-padding) * 2);
+        padding-left: var(--table-padding);
+        font-weight: 700;
+        text-transform: uppercase;
+        letter-spacing: 2px;
+        font-size: 0.75rem;
+
+        &[align="center"] {
+          text-align: center;
+        }
+
+        &[align="right"] {
+          text-align: right;
+        }
+
+        &:first-of-type {
+          padding-left: calc(var(--table-padding) * 2);
+        }
+
+        &:last-of-type {
+          padding-right: var(--table-padding);
+        }
+
+        a {
+          color: var(--table-header-foreground);
+          font-weight: 700;
+        }
+      }
+    }
+  }
+
+  & > tbody {
+    & > tr {
+      background: var(--table-body-background);
+      color: var(--table-body-foreground);
+      border-bottom: 2px solid var(--table-background);
+
+      & > td {
+        vertical-align: middle;
+        padding: var(--table-padding) 0 var(--table-padding) var(--table-padding);
+        font-weight: 500;
+        height: 50px;
```

Comment:
This leads to _any_ table using a height of 50 px per row, while you probably wanted this only for the table listing the Jenkins jobs. For instance the test results table looks horrible with this huge space now.

Reply:
Please file an appropriate issue labeled as regression on Jira (https://issues.jenkins.io/secure/Dashboard.jspa) including steps to reproduce and screenshots from the old and current behavior.